api/routes.go

```diff
@@ -3,24 +3,28 @@ package api
 import (
 	"encoding/hex"
 	"errors"
-	"github.com/gorilla/mux"
-	"github.com/kailt/imageresizer/etag"
-	"github.com/kailt/imageresizer/imager"
-	"github.com/rcrowley/go-metrics"
+	"fmt"
 	"io"
 	"io/ioutil"
 	"net/http"
 	"os"
 	"strconv"
 	"strings"
 	"unicode/utf8"
```

Comment:
The file was already `gofmt`ed. This reorder should have been a separate commit because it's a cosmetic change unrelated to the main PR.